scripts/migration/lib.sh

```diff
@@ -129,6 +129,20 @@ config_validate() {
           pass="false"
       fi
     done
+
+    sync_enabled=$(yq4 '.objectStorage.sync.enabled' <<< "${CONFIG["${1}"]}")
+    sync_default_enabled=$(yq4 '.objectStorage.sync.syncDefaultBuckets' <<< "${CONFIG["${1}"]}")
+    if [[ "${1}" = "sc" ]] && [[ "${sync_enabled}" = "true" ]] && [[ "${sync_default_enabled}" = "true" ]]; then
+      log_info "checking sync swift"
+
+      check_harbor="$(yq4 '.harbor.persistence.type' <<< "${CONFIG["${1}"]}")"
+      check_thanos="$(yq4 '.thanos.objectStorage.type' <<< "${CONFIG["${1}"]}")"
+      check_sync_swift="$(yq4 '.objectStorage.sync.swift' <<< "${CONFIG["${1}"]}")"
+
+      if { [[ "${check_harbor}" = "swift" ]] || [[ "${check_thanos}" = "swift" ]]; } && [[ "${check_sync_swift}" = "null" ]]; then
+        log_error "error: swift is enabled for Harbor/Thanos, but .objectStorage.sync is missing swift configuration"
+      fi
+    fi
```

Comment:
Is this something we want to add here as well? https://github.com/elastisys/compliantkubernetes-apps/blob/14a740c8995826b265609b496bac70cdd7cb913d/bin/common.bash#L337

Reply:
Or maybe that is not needed with the checks you have in the helm templates.

Reply:
yes, there were already checks in the helm files

Reply:
Ok, then let's skip this in bin/common.bash